packages/@vue/cli-plugin-e2e-nightwatch/package.json

```diff
@@ -26,11 +26,12 @@
     "@vue/cli-shared-utils": "^4.0.0-rc.1",
     "deepmerge": "^3.2.0",
     "execa": "^1.0.0",
-    "nightwatch": "^1.1.11",
-    "selenium-server": "^3.141.59"
+    "nightwatch": "^1.2.2",
+    "chromedriver": "^76.0.1",
```

Comment:
For now please make `chromedriver` a peer dependency. It bumps its major version every 6 weeks and Vue CLI certainly cannot do major releases at that pace. So it's better to be managed on the user side.
In later RC (or minor) versions, I'd like to introduce a preflight check, and a prompt interface for users to choose which version of browser drivers they need.